data/mods/gen3/scripts.js

```diff
@@ -6,7 +6,8 @@ let BattleScripts = {
 	gen: 3,
 	init() {
 		for (let i in this.data.Pokedex) {
-			delete this.data.Pokedex[i].abilities['H'];
```

Comment:
I assume this means you tested and found this unnecessary?

Reply:
yes